utility/src/main/java/org/oppia/android/util/parser/image/UrlImageParser.kt

```diff
@@ -121,11 +123,18 @@ class UrlImageParser private constructor(
         htmlContentTextView.width { viewWidth ->
           proxyDrawable.initialize(drawable, computeBounds(drawable, viewWidth))
           htmlContentTextView.text = htmlContentTextView.text
+          if(isRtlLayout(htmlContentTextView)){
+            htmlContentTextView.textDirection = View.TEXT_DIRECTION_ANY_RTL
```

Comment:
Could we set this in HtmlParser like we do the movement method?

Reply:
Done.